staff/sergio-costa/e-commerce-template/style.css

```diff
@@ -0,0 +1,148 @@
+body{
+    margin:0;
+    height: 100vh;
+    padding: 0;
+    background-color: white;
+}
+
+header{
+    position: fixed;
+    background-color: lightgreen;
+    /* z-index: 1; */
+    width: 100%;
+    height: 150px;
+    top:0;
+    display: flex;
+    align-items: center;
+    justify-content:space-around;
+}
+
+main{
+    margin-top:175px;
```

Comment:
You could play with position relative and absolute respect body the element and then fix the top of this element to 175px.

Reply:
👍